core/src/state.rs

```diff
@@ -10,6 +10,8 @@ use std::{
     sync::{mpsc::Sender, Arc},
 };
 
+/// The Store of the Holochain instance Object, according to Redux pattern.
+/// Its composed of a Nucleus and Agent state slice.
```

Comment:
I'd rather say: "It's composed of all sub-module's state slices. To plug in a new module its state slice needs to be added here."

Reply:
Fixed. Please approve PR :)